.gitignore

```diff
@@ -104,3 +104,11 @@ venv.bak/
 .mypy_cache/
 
 *.tar.gz
+
+# just some dev files
+data
```

Comment:
These should be in a global ignore for your environment. The dr_util_config.json can stay though

Reply:
Done